src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  background-color: #f4f4f4;
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo-link {
+  display: flex;
+}
+
+.header__nav-list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.header__nav-list .header__nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.header__nav_link {
```

Comment:
The class name `.header__nav_link` seems to be a typo. It should follow the BEM naming convention like other class names in the file, which would be `.header__nav-link`.